hosts

```diff
@@ -5,6 +5,12 @@ stats.galaxyproject.eu
 [build]
 build.galaxyproject.eu ansible_ssh_user=root
 
+[beacon]
+beacon-1.galaxyproject.eu ansible_ssh_user=rocky
+
+[beacon_import]
+beacon-import.galaxyproject.eu ansible_ssh_user=rocky
```

Comment:
DNS entry is different, https://github.com/usegalaxy-eu/infrastructure/blob/a71d5dbfc08591a43d9e45ba050511db91d53025/instance_dedicated_beacon_import.tf#L33

Reply:
yes I have to update this, thank you